bower.json

```diff
@@ -29,7 +29,7 @@
     "c3": "0.4.11",
     "kubernetes-label-selector": "2.0.0",
     "kubernetes-container-terminal": "2.1.0",
-    "registry-image-widgets": "0.0.11",
+    "registry-image-widgets": "https://github.com/cockpit-project/registry-image-widgets.git#0.0.13",
```

Comment:
I'm not sure what's involved in this, but is it possible to update bower to use the new repository so we don't need the full URL?